### PR TITLE
Dialog Service

### PR DESCRIPTION
**Pull request by DamianSuess**
IDialogService for v8.1.97

* [X] Implement DialogService
* [X] Create sample projects
* [X] Cleanup code

**Ready for review**

### CODE REVIEW

**Comment by DamianSuess** [comment]
Still need to cleanup the base window style to auto-adjust to the child content and clean-up the sample app

**Comment by DamianSuess** [comment]
@OmidID, the PR is ready for review. We got IDialogService! :)

File: samples/SampleDialogApp/.gitignore [hunk 1]
```diff
@@ -0,0 +1,454 @@
+## Ignore Visual Studio temporary files, build results, and
```
Comment (DamianSuess) [inline samples/SampleDialogApp/.gitignore:1]: Remove this file

**Comment by DamianSuess** [comment]
With this feature nailed down, it's a great candidate for the NuGet package

**Comment by DamianSuess** [comment]
Thank you!

**Comment by OmidID** [comment]
Maybe is the time to publish nuget packages.

**Comment by DamianSuess** [comment]
indeed my man. Do you want me to take care of this one, or do you got it?

**Comment by OmidID** [comment]
@DamianSuess Sorry. I was busy. I think it is late. Already you did it :D

**Comment by DamianSuess** [comment]
@OmidID no worries, I completely understand - _we all got bills to pay_ 😆
I go so wrapped up in testing the new infrastructure, I forgot to reply back on this thread to let you know.

**Pull request merged**